### DISCUSSION TITLE
yeelights stopped working with hyperHDR 20 / Deb12

### DISCUSSION THREAD

**Discussion by pho084** [awawa-dev/HyperHDR#665] [General]
Hi awawa
first of all great work on hyperhdr 20 as per usual it runs /looks great. thanks!

however i upgraded my RPI4 running raspbian bullseye to native Debian 12 64bit. I imported the old config just fine and everything _seemed_ to be working... except the yeelights. They do light up when hyperion is (re)started but they're stuck in blue color.

logs show succesful connection but at some point just throw the error "all yeelights are in error stopping yeelights" or something.

so i was wondering if there were any breaking changes w the new version regarding the yeelights?

**Comment by awawa-dev** [comment]
no

**Comment by pho084** [comment]
ok ty. 
i completely removed them from network and factory resetted. added them again to the network and ran the wizard. they are discovered, they light up when probing in the wizard.
but again they're stuck after that. remote control changing colors does not do anything. 
i can turn them on and off though

**Comment by awawa-dev** [comment]
please attach the full HyperHDR logs from the connection attempt
